mne/utils/numerics.py

```diff
@@ -563,11 +563,13 @@ def grand_average(all_inst, interpolate_bads=True, drop_bads=True):
 
     Raises
     ------
-    If ``all_inst`` is empty, or if not all elements are evokeds.
+    ValueError
+        If ``all_inst`` is empty, or if not all elements are evokeds.
```

Comment:
should we be doing `Raises` and `Warns` docstring sections on all functions/methods?  I've been assuming we were following the convention of "only document warnings/errors in the docstring if it raises a MNE-Python-specific warning/error class" (which I don't think we define anyway?)

Reply:
Agreed, I wasn't too keen on adding this in #7456 either, as we do raise lots of warnings and errors in lots of places. I'll just remove it for now and we can add it back later if we agree on a plan.